Editor/Window/RegisterComputeInput.cs

```diff
@@ -114,6 +118,49 @@ private async void OnRegisterComputeButtonClicked()
                         Strings.ViewLogsStatusBoxUrlTextButton);
                 }
             }
+            
+            UpdateGUI();
+        }
+
+        private async void CheckCompute()
+        {
+            if (_computeState is ComputeStatus.Registered)
+            {
+                var computes = await _stateManager.ComputeManager.ListCompute(_stateManager.AnywhereFleetId);
+                if (computes.Success)
+                {
+                    var matchingCompute =
+                        computes.ComputeList.FirstOrDefault(compute =>
+                            compute.ComputeName == _stateManager.ComputeName);
+                    if (matchingCompute == null)
+                    {
+                        _computeNameInput.value = _defaultComputeName;
+                        var defaultIpAddress = _defaultIpAddress.Split(".");
+                        for (var i = 0; i < defaultIpAddress.Length; i++)
+                        {
+                            _ipInputs[i].value = defaultIpAddress[i];
+                        }
+                        _computeState = ComputeStatus.NotRegistered;
+                    }
+                    else
+                    {
+                        _stateManager.ComputeName = matchingCompute.ComputeName;
+                        _stateManager.IpAddress = matchingCompute.IpAddress;
+                        _stateManager.WebSocketUrl = matchingCompute.GameLiftServiceSdkEndpoint;
+
+                        var ipAddress = matchingCompute.IpAddress.Split(".");
+                        for (var i = 0; i < ipAddress.Length; i++)
+                        {
+                            _ipInputs[i].value = ipAddress[i];
+                        }
+
+                    }
+                }
+                else
+                {
+                    _computeState = ComputeStatus.NotRegistered;
```

Comment:
We should also do a Debug.LogError with the error response message

Reply:
This switch to NotRegistered should not happen since the stateManager indicates that it is registered. If you display an error response, the user can take the corresponding action to ensure it works the next time around.

Reply:
Fixed